manifests/federation/0.0.5/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -6,17 +6,26 @@ metadata:
   name: federation.v0.0.5
   namespace: placeholder
   annotations:
+    capabilities: Basic Install
     categories: "OpenShift Optional, Integration & Delivery"
-    description: "Kubernetes Federation namespace-scoped installation"
+    description: Gain Hybrid Cloud capabilities between your clusters with Kubernetes Federation.
+    certified: false
+    containerImage: "quay.io/openshift/origin-federation-controller"
```

Comment:
It's good practice for CSVs to use tagged images, since this can be updated in subsequent CSVs